example/src/App.js

```diff
@@ -64,7 +65,7 @@ export default function App() {
         <Button
           title="Select video"
           onPress={() =>
-            ImagePicker.launchImageLibrary({mediaType: 'video'}, (response) => {
+            ImagePicker.launchImageLibrary({mediaType: 'video', saveToPhotos: true}, (response) => {
```

Comment:
The save seems to work when I take new videos/photos, which is great.
I'm assuming it's not needed for when selecting an image/video, as suggested by this example?

Reply:
Yes, it was my mistake. Can you revert this change and merge?